### PR TITLE
smart monitoring - test drives directly

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

**Comment by szaimen** [comment]
This was successfully tested 👍

File: disk/smart-monitoring.sh [hunk 1]
```diff
@@ -54,6 +62,37 @@ fi
...
+# Test drives
+print_text_in_color "$ICyan" "Testing if all drives support smart monitoring and are healthy..."
+mapfile -t DRIVES <<< "$DRIVES"
+for drive in "${DRIVES[@]}"
+do
+    echo '#########################'
+    print_text_in_color "$ICyan" "Testing /dev/$drive"
+    if ! smartctl -a "/dev/$drive"
+    then
+        print_text_in_color "$IRed" "/dev/$drive doesn't support smart monitoring or isn't healthy"
```
Comment (enoch85) [inline disk/smart-monitoring.sh:75]: is this intended? first print then msg_box?

Comment (szaimen) [reply disk/smart-monitoring.sh:75]: yes, for the output log of the script this makes it much more readable. But I also think that showing the msg_box here makes sense -> I do both

Comment (szaimen) [reply disk/smart-monitoring.sh:75]: I could also reverse the order -> show msg_box first and then print_text_in_color. What do you think?

Comment (enoch85) [reply disk/smart-monitoring.sh:75]: Whatever. Doesn't matter to me.

**Comment by enoch85** [comment]
So, what does this do exactly?

It seems to test the disks somehow?

**Comment by szaimen** [comment]
> So, what does this do exactly?
> 
> It seems to test the disks somehow?

Yes, it tests all connected drives directly and reports if a drive is unhealthy/doesn't support smart monitoring so that you know directly what is up with your drives :)

**Pull request merged**